examples/resources/port_scorecard/main.tf

```diff
@@ -0,0 +1,18 @@
+resource "port_scorecard" "production_readiness" {
+  identifier = "production-readiness"
+  title      = "Production Readiness"
+  blueprint  = "microservice"
+  rules = [{
+    identifier = "high-avalability"
+    title      = "High Availability"
+    level      = "Gold"
+    query = {
+      combinator = "and"
+      conditions = [{
+        property = "replicaCount"
+        operator = ">="
+        "value"  = "4"
```

Comment:
Should value have quotes? should the number have quotes?

Reply:
Currently, In terraform I only support value of string, I need to add support of  number value  and boolean value